src/resources/batches/BatchList.tsx

```diff
@@ -68,7 +68,7 @@ const PlatformFilter = (props: PlatformFilterType): React.ReactElement => {
   const { data } = useGetMany(reference, { ids: platformIds })
   useEffect(() => {
     if (data != null) {
```

Comment:
Hello @TahaKhanAbdalli - the `ActivePlatform` filter wasn't working for me.  I changed the test to use a _logical_ true value, rather than an explicit `true` - and it worked.

Are you ok with it?